frontend/src/routes/AddSuggestionPage.jsx

```diff
@@ -0,0 +1,93 @@
+//UI imports
+import React, {Component} from 'react';
+import {Table, Label, PageHeader, Button, ButtonGroup, Modal, Panel} from 'react-bootstrap';
+import PropTypes from 'prop-types';
+import SongForm from '@Components/SongForm.jsx';
+
+//Data imports
+import {Container} from 'flux/utils';
+import Song from '@Models/Song.js';
+import SongsStore from '@Stores/SongsStore.js';
+
+//Interaction imports
+import {addSongAction} from '@Actions/SongActions.js';
+import {deferredDispatch, dispatch} from '@Services/AppDispatcher.js';
+
+class AddSuggestionPage extends Component{
+    constructor(props){
+        super(props);
+        this.handleNewSong = this.handleNewSong.bind(this);
+
+        this.state = {
+            modal: false
+        }
+
+        //Bound callback
+        this.clearModal = this.clearModal.bind(this);
+    }
+    clearModal(){
+        this.setState({modal:false});
+    }
+    /**
+     * Handles creation of a new suggestion
+     * @param {Song} song The song object that was created 
+     */
+    handleNewSong(song){
+        deferredDispatch(addSongAction(song.toJS()));
+        let modal = {
+            body: ( <React.Fragment>
+                    <p>Je suggestie is nu toegevoegt aan de suggestielijst, de commissie zal er bij de volgende vergadering naar luisteren!</p>
```

Comment:
'toegevoegd' :-P

Reply:
Menu item is er, suggestielijst zelf is idd nog niet geimplementeerd. En spelling is niet mijn ding xD

Reply:
Ah, helemaal dat menu item over het hoofd gezien :-P